RecoLocalCalo/HcalRecAlgos/src/MahiFit.cc

```diff
@@ -546,16 +544,18 @@ void MahiFit::phase1Debug(const HBHEChannelInfo& channelData, MahiDebugInfo& mdi
         mdi.itPulse[iTS] = nnlsWork_.pulseMat.col(iBX).coeff(iTS);
       }
     } else if (nnlsWork_.bxs.coeff(iBX) == pedestalBX_) {
+      mdi.pedEnergy = 0;
       mdi.pedEnergy = nnlsWork_.ampVec.coeff(iBX);
```

Comment:
L547 is unnecessary. 
Was it intentional, or was something else supposed to be assigned to 0?

Reply:
L547 is removed.

Reply:
L547 is removed